components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -1258,16 +1294,26 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
 !
 ! Aerosol processes ...
 !
-
     if (uci1_ndx > 0) then
        vmr_old2(:ncol,:,:) = vmr(:ncol,:,:)
     endif
+
     call t_startf('aero_model_gasaerexch')
-    call aero_model_gasaerexch( imozart-1, ncol, lchnk, delt, latndx, lonndx, reaction_rates, &
-                                tfld, pmid, pdel, mbar, relhum, &
-                                zm,  qh2o, cwat, cldfr, ncldwtr, &
-                                invariants(:,:,indexm), invariants, del_h2so4_gasprod,  &
-                                vmr0, vmr, pbuf )
+#if (defined MODAL_AERO_5MODE | defined MODAL_AERO_4MODE_MOM | defined MODAL_AERO_4MODE | defined MODAL_AERO_9MODE | defined MODAL_AERO_7MODE)  
+       call aero_model_gasaerexch( imozart-1, ncol, lchnk, delt, latndx, lonndx, reaction_rates, &
+                                tfld, pmid, pdel, mbar, relhum,                               &
+                                zm,  qh2o, cwat, cldfr, ncldwtr,                              &
+                                invariants(:,:,indexm), invariants, del_h2so4_gasprod,        &
+                                vmr0, vmr, pbuf, troplev)
+#else
+       ! this option is left for non modal aerosol
+       call aero_model_gasaerexch( imozart-1, ncol, lchnk, delt, latndx, lonndx, reaction_rates, &
+                                tfld, pmid, pdel, mbar, relhum,                               &
+                                zm,  qh2o, cwat, cldfr, ncldwtr,                              &
+                                invariants(:,:,indexm), invariants, del_h2so4_gasprod,        &
+                                vmr0, vmr, pbuf)
+#endif 
```

Comment:
I know it works, It would be better to use || like other places instead of |?

Reply:
thanks. corrected